kolibri/plugins/device/assets/src/views/__test__/RearrangeChannelsPage.spec.js

```diff
@@ -1,12 +1,12 @@
 import { shallowMount } from '@vue/test-utils';
-import useUser, { useUserMock } from 'kolibri.coreVue.composables.useUser';
-import useSnackbar, { useSnackbarMock } from 'kolibri.coreVue.composables.useSnackbar';
+import useUser, { useUserMock } from 'kolibri/composables/useUser'; // eslint-disable-line
+import useSnackbar, { useSnackbarMock } from 'kolibri/composables/useSnackbar'; // eslint-disable-line
```

Comment:
Why does eslint not like these lines?

Reply:
Because the named exports only exist inside the mock, not inside the composable module itself, but because it is being mocked, they are available as an export.